lib/DOMFetcher.js

```diff
@@ -0,0 +1,62 @@
+/* globals Services */
+"use strict";
+
+const {Cu} = require("chrome");
+Cu.import("resource://gre/modules/Services.jsm");
+
+/**
+ * An event listener that will capture the DOM
+ * content of every closing HTMLDocument.
+ */
+function DOMFetcher() {
+  this.listeners = new Set();
+  Services.mm.loadFrameScript("resource://loci/data/content/DocumentChangeMessenger.js", true);
```

Comment:
`DocumentChangeManager` is currently in `content-src/static`. Why not put it directly in `data`?

The `content-src` directory is reserved for the UI. `data` would be more appropriate, where `content-bridge.js` is located.